dt-metrics/combined/time-charts.php

```diff
@@ -0,0 +1,113 @@
+<?php
+if ( !defined( 'ABSPATH' ) ) {
+    exit;
+} // Exit if accessed directly.
+
+class DT_Metrics_Time_Charts extends DT_Metrics_Chart_Base
+{
+
+    //slug and title of the top menu folder
+    public $base_slug = 'combined'; // lowercase
+    public $base_title;
+    public $title;
+    public $slug = 'time_charts'; // lowercase
+    public $js_object_name = 'wp_js_object'; // This object will be loaded into the metrics.js file by the wp_localize_script.
+    public $js_file_name = '/dt-metrics/combined/time-charts.js'; // should be full file name plus extension
+    public $permissions = [ 'access_contacts', 'view_project_metrics' ];
+    public $post_types = [];
+    public $field_settings = [];
+    public $post_type_select_options = [];
+    public $post_field_select_options = [];
+    public $post_field_types_filter = [ 'date', /* 'key_select', 'multi_select', */ /* 'connection', 'number' */ ]; // connection and number would be interesting for additions to groups, and quick button usage
+
+    public function __construct() {
+        parent::__construct();
+        if ( !$this->has_permission() ){
+            return;
+        }
+
+        $this->title = __( 'Time Charts', 'disciple_tools' );
+        $this->base_title = __( 'Project', 'disciple_tools' );
+
+        $url_path = dt_get_url_path();
+        if ( "metrics/$this->base_slug/$this->slug" === $url_path ) {
+
+            add_action( 'wp_enqueue_scripts', [ $this, 'scripts' ], 99 );
+
+        }
+
+        // $post_types = DT_Posts::get_post_types(); // This is only giving [ 'peoplegroups' ] at the moment
+        $post_types = [ 'contacts', 'groups', 'peoplegroups' ];
```

Comment:
@corsacca this was the bit of code put in to enable me to continue, but the line above was not giving the full list of post types :(

Reply:
@squigglybob no problem, i'll tackle that one.